src/pool/srv_target.c

```diff
@@ -1263,9 +1247,13 @@ ds_pool_stop(uuid_t uuid)
 
 	ds_rebuild_abort(pool->sp_uuid, -1, -1, -1);
 	ds_migrate_stop(pool, -1, -1);
+
+	pool_stop_children(pool);
```

Comment:
Looks it needs be called on error cleanup in ds_pool_start(). (and in pool_alloc_ref() when the collective call partially failed?).

Reply:
Indeed... Will fix. Thanks, Niu.